src/facet_display/SearchFilter.test.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import upperCase from "lodash.uppercase"
```

Comment:
Is `upperCase` only being used in a test? I would drop it and use `String.prototype.toUpperCase`